terraform.tf

```
@@ -0,0 +1,19 @@
+terraform {
+  /* Uncomment this block to use Terraform Cloud for this tutorial
+	cloud {
+			organization = "organization-name"
+			workspaces {
+				name = "learn-terraform-outputs"
+			}
+	}
+	*/
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.43.0"
+    }
+  }
+
+  required_version = "~> 1.2.0"
```

```suggestion
  required_version = ">= 1.2.0"
```

I _think_ this is what's intended since it's what shown in the tutorial, but I'm realizing I'm not 100% sure what we standardize on, so I'll defer to you :)